app/src/main/java/org/jellyfin/androidtv/util/apiclient/BaseItemUtils.kt

```diff
@@ -0,0 +1,231 @@
+package org.jellyfin.androidtv.util.apiclient
```

Comment:
It looks like you chose to convert the existing Java file to Kotlin. In that case you should do the renaming of the file in a separate commit so the git history remembers the old file. It also means I'll review all the Kotlin code since we have higher standard for it.

Reply:
I deleted the commits and did the renaming in a second commit, but it still removed the git history. I'm not sure how to rename the file from .java to .kt and have the git history remember the old file. 

The rest of the comments have been addressed.

Reply:
It might not show it in GitHub but the history is now properly tracked